src/AddIns/Uno.UI.MediaPlayer.Skia.Gtk/GTKMediaPlayer.events.cs

```diff
@@ -289,6 +294,7 @@ private static void OnSourceChanged(DependencyObject source, DependencyPropertyC
 		if (source is GtkMediaPlayer player && args.NewValue is string encodedSource)
 		{
 			player._updateVideoSizeOnFirstTimeStamp = true;
+			player._isParsedLocalFile = false;
 
 			if (typeof(GtkMediaPlayer).Log().IsEnabled(Microsoft.Extensions.Logging.LogLevel.Debug))
 			{
```

Comment:
Let's move that whole method to be a instance method, so we avoid all the `player._` statements